lib/Fmt_ast.ml

```diff
@@ -2907,7 +2918,7 @@ and fmt_value_description c ctx vd =
   @@ fun c ->
   let pre = if List.is_empty pval_prim then "val" else "external" in
   let doc_before, doc_after, atrs =
-    fmt_docstring_around_item c pval_attributes
+    fmt_docstring_around_item ~is_val:true c pval_attributes
```

Comment:
This applies `doc-comments-val` on `external`s in structures. Is this a problem ?

Reply:
It is what I would expect, since they are similar to vals, what do you think @craigfe ?

Reply:
It's also what I would expect, and I think the documentation makes this sufficiently clear.